classes/mender-install.bbclass

```diff
@@ -1,6 +1,10 @@
 PREFERRED_VERSION_go_cross = "1.6%"
 
-#Add support for systemd
+# Add support for systemd
+#
+# Note that a very common error with systemd is that the system hangs during the
+# boot process while looking for devices. This is almost always because the
+# kernel feature CONFIG_FHANDLE is not enabled.
 DISTRO_FEATURES_append = " systemd"
```

Comment:
Yes, agree, switching to systemd shouldn't be done behind your back. We decided that we didn't wanna deal with sysv scripts (this was backed by Eystein's field surveys), so I propose we just produce an error then. Do you know of a mechanism for this in Yocto, or should we simply test the `DISTRO_FEATURES` variable in a task snippet?